Feature/members page

Megumikawa opened the pull request:
PCとSP両方ほぼ完成したのでご確認をお願いします。

ただ、未完成な部分が数点あります。
1) SPの時に右側にスペースが邪魔しているのを探してるんですが未だに原因が見つかってないです。
2) members-page__main-imageクラスのイメージのサイズがどうも合わず白色のブランクができてしまいます。
3) members-partnerクラスのバックグラウンドを色指定のものに変更したんですが、お問い合わせとフッターの部分も色を引き継いでしまいます。

以上、３点がまだ未解決です。

Megumikawa committed:
test

Megumikawa committed:
Merge remote-tracking branch 'origin/develop' into feature/members-page

Megumikawa committed:
merge conflict resolved

Megumikawa committed:
created top and main image in members page

Megumikawa committed:
added simplely HTML base of members page

Megumikawa committed:
add each class name

Megumikawa committed:
Added style of member page main part

Megumikawa committed:
created style of members-page's partner part

Megumikawa committed:
Finised of PC style

Megumikawa committed:
start styling of responsive until image

Megumikawa committed:
Merge remote-tracking branch 'origin/develop' into feature/members-page

Megumikawa committed:
fix partner area flex for responsive

Megumikawa committed:
almost finished of responsive

Megumikawa committed:
change member-page-main dus to flex position

Megumikawa committed:
fix indent and delete comment out

MomokaIgarashi commented on home.php:
```diff
@@ -1,6 +1,131 @@
+            </ul>
+        <div>
+    </div>
 
```
>3 members-partnerクラスのバックグラウンドを色指定のものに変更したんですが、お問い合わせとフッターの部分も色を引き継いでしまいます。

この問題はdivのタグ閉じ忘れなので、129行目に</div>を追加すれば解決します

MomokaIgarashi committed:
Update header.php

＞１：SPの時に右側にスペースが邪魔しているのを探してるんですが未だに原因が見つかってないです。

19行目に<div class="body-wrap">を入れ、over flow hiddenをかければ解決します！
閉じタグはfooterに記述します

MomokaIgarashi committed:
Update home.php

閉じタグ追加

MomokaIgarashi committed:
Update members.css

SPの右余白解決記述

.body-wrap{
    overflow: hidden;
}

MomokaIgarashi committed:
Update footer.php

閉じタグ追加
